mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxPath.java

```diff
@@ -178,14 +178,4 @@ public final int hashCode() {
         return Objects.hash(namespace, user, name);
     }
 
-    /**
-     * Return the full name of the {@link MailboxPath}, which is constructed via the {@link #namespace} and {@link #name}
-     * 
-     * @param delimiter
-     * @return fullName
-     */
-    public String getFullName(char delimiter) {
```

Comment:
no need to deprecate this at first?

Reply:
Could be if you prefer. IMO we never refrained ourselves from breaking changes within the mailbox/api so far...

Reply:
Yes, I'm not sure, but MailboxPath is a very broadly used one...

Reply:
Ok for only deprecating this...

Reply:
I would have remove it too